src/rgb/transition.rs

```diff
@@ -15,6 +15,7 @@
 use super::{State, Metadata, Script};
 
 pub struct Transition {
+    pub id: usize,
```

Comment:
Same as above: we have to avoid non-statically-sized types (like platform-dependend)